src/components/Form.js

```diff
@@ -66,24 +79,38 @@ export default function Form({ handleActivity, preloadedValues, title }) {
         notes: data.notes,
         date: data.date,
         location: data.location,
+        photo: photo,
       });
       navigate(`/`);
+      setPhoto('');
     }
   };
 
   useEffect(() => {
     setFocus('activity');
   }, [setFocus]);
 
+  function onDeletePicture(event) {
+    event.preventDefault();
+    setPhoto('');
+  }
+  const [preloadedPicture, setPreloadedPicture] = useState(
+    preloadedValues?.photo
+  );
+  function onDeletePreloadedPicture(event) {
+    event.preventDefault();
+    setPreloadedPicture('');
+  }
+
```

Comment:
maybe it would be better to declare the function under the returned part to get a quicker view of the output of the component without scrolling so much over all the different functions :)